blockchain/integrity/integrity_test.go

```diff
@@ -1261,12 +1261,14 @@ func TestLoadBlockchainfromDB(t *testing.T) {
 		fmt.Printf("Cannot add block 3 again: %v\n", err)
 
 		// invalid address returns error
-		act, err := accountutil.AccountState(sf, "")
```

Comment:
@dustinxie what to put here to make this test pass ?

Reply:
`_, err = address.FromString("")`

Reply:
done